hw-probe.pl

```diff
@@ -217,15 +217,19 @@
 
 if($#ARGV_COPY==-1)
 { # Run from STDIN
-    print "Executing hw-probe -all -upload\n\n";
-    $Opt{"All"} = 1;
-    $Opt{"Upload"} = 1;
+    if (-t STDIN) {
+       	print "Executing hw-probe -all\n\n";
+        $Opt{"All"} = 1;
+    } else {
+       	print "Executing hw-probe -all -upload\n\n";
+       	$Opt{"All"} = 1;
+       	$Opt{"Upload"} = 1;
+    }
 }
 elsif($#ARGV_COPY==0 and grep { $ARGV_COPY[0] eq $_ } ("-snap", "-flatpak"))
 { # Run by desktop file
-    print "Executing hw-probe -all -upload\n\n";
```

Comment:
This is for a "Launch" buttons in Flatpak and Snap stores. This should not be changed.

Reply:
Reverted.